src/core/function_messages_cqrs.tf

```diff
@@ -84,11 +95,9 @@ module "function_messages_cqrs" {
     "private_dns_zone_blob_ids"  = [data.azurerm_private_dns_zone.privatelink_blob_core_windows_net.id],
     "private_dns_zone_queue_ids" = [data.azurerm_private_dns_zone.privatelink_queue_core_windows_net.id],
     "private_dns_zone_table_ids" = [data.azurerm_private_dns_zone.privatelink_table_core_windows_net.id],
-    "queues" = [
-      local.function_messages_cqrs.app_settings.MESSAGE_VIEW_UPDATE_FAILURE_QUEUE_NAME
-    ],
```

Comment:
why this? module.function_messages_cqrs.storage_account.name is a public storage used for function start-up.

Reply:
fixed, thank you!